pixart.js

```diff
@@ -1 +1,57 @@
+// var submitForm = document.getElementById("color-field");
+// var form = $("form");
+// var box = document.querySelector(".brush");
+// var setColor= document.getElementById("set-color");
+// // $("#set-color");
+// ///
+// $("form").click(function(event){
+//     event.preventDefault();
+// });
+//
+//
+// box.addEventListener("click", handleClickEvent);
+//
+// function handleClickEvent() {
+// // submitForm.innerHTML = submitForm;
+//   // box.innerHTML = submitForm;
+//   box.style.backgroundColor = form;
+// }
+// ///
+// submitForm.addEventListener("submit", handleSubmitEvent);
+//
+// function handleSubmitEvent() {
+//   box.innerHTML=submitForm;
+// }
 
+//set up variables
+var form = $("form");
+var input = document.querySelector('#color-field');
+var brush = document.querySelector('.brush');
+
+
+$("form").click(function(event){
+    event.preventDefault();
+  brush.style.backgroundColor = input.value;
+});
+
+
+for (var i = 1; i<=8000; i++) {
+  var newDiv = document.createElement("div");
```

Comment:
This could be shortened using jQuery, but I realize that may have been a result of the unclear / conflicting instructions on the exercise.